Makefile

```diff
@@ -6,11 +6,23 @@
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+init: ## Initialize the project
+	make install
+	make build
+
 install: ## Install the dependencies
-	npm install
+	cd curator && npm install
+	cd website && npm install
+
+webpage: ## Run the webpage localy
+	cd website && npm run start
 
 build: ## Compile the project
-	npm build
+	cd curator && npm run build
+	cd website && npm run build
 
 run: ## Summarize a list of articles
-	npm start
\ No newline at end of file
+	cd curator && npm start
```

Comment:
**suggestion:** Can you setup a docker compose to start the apps ?

Reply:
For the next steps for sure but just right now this part could be enough like that IMO